lms/templates/verify_student/pay_and_verify.html

```diff
@@ -65,7 +66,7 @@
       data-platform-name='${platform_name}'
       data-course-key='${course_key}'
       data-course-name='${course.display_name}'
-      data-course-start-date='${course.start_datetime_text()}'
+      data-course-start-date='${course_start_datetime_text(course.start)}'
```

Comment:
My question is more getting at the possible regression in functionality with this change.  Since the advertised data is no longer being sent, it won't be used when a course has it specified but not a timestamp.